ingeniamotion/pdo.py

```diff
@@ -98,9 +88,14 @@ def data(self) -> Tuple[Deque[float], List[Deque[Union[int, float]]]]:
             the readings values.
 
         """
-        data = deepcopy(self.__buffer)
-        time_stamps = deepcopy(self.__timestamps)
-        self._clear_buffers()
+        time_stamps = []
```

Comment:
Will motionlab have to reprocess this data?
If you think the data would be better presented to it in the buffer form please create an issue.
We could potentially leave this for retrocompatibility but improve the performance of the scope

Reply:
I kept compatibility with the other Poller and Monitoring, so the scope receives the data in similar formats. But we could consider improving it. We need to improve the other poller as well.